### DISCUSSION TITLE
Demo's source is not supported

### DISCUSSION THREAD

**Discussion by intefa** [akiver/cs-demo-manager#663] [Q&A] [answered]
![1](https://github.com/akiver/cs-demo-manager/assets/121894383/d1ac2e51-8eac-4c71-b810-c7c0fdcaa967)

**Comment by akiver** [comment]
The "**Source**" column shows a `?`  which means that the source of the demos is unrecognized.
I guess they come from a private server? 
Please read this [doc](https://cs-demo-manager.com/docs/faq/stats#what-are-the-supported-demo-sources) - you can change the source (right-click on the demos) and analyze them, but if the demos come from a non supported source it may not work correctly.

**Reply by intefa** [reply, marked as answer]
After reading your tips, I studied it, changed the source to Valve, and then I can analyze it normally. Thank you for developing this software, which makes it easier for me to edit videos
